frontend/src/modules/teamcomments/components/TeamComments.js

```diff
@@ -36,14 +36,14 @@ export default function TeamComments(props: Props) {
                     </Localized>
                 </div>
                 :
-                <div className='comments-list'>
-                    <AddComment
-                        user={ user.nameOrEmail }
-                        username={ user.username }
-                        imageURL={ user.gravatarURLSmall}
+                <section className="team-comments">
+                    <CommentsList
+                        comments={ comments }
+                        user={ user }
+                        canComment={ canComment }
                         addComment={ addComment }
                     />
-                </div>
+                </section>
```

Comment:
This entire `section.team-comments` is repeated below. We should only have it once. You can achieve that by changing the condition.